engine/flink/management/periodic/src/main/resources/db/batch_periodic/migration/hsql/V1_104__add_schedule_name.sql

```diff
@@ -0,0 +1 @@
+ALTER TABLE "periodic_process_deployments" ADD COLUMN "schedule_name" VARCHAR(16777216) default 'schedule' not null ;
```

Comment:
1. It's the same as postgresql? So we can put it in default migration?
2. Do we realy need there `VARCHAR(16777216)` It looks like 255 / 512 ?

Reply:
I changed it...